src/extensions/default/bramble/stylesheets/consoleTheme.css

```diff
@@ -23,10 +23,10 @@
     display:block;
     font-size: inherit;
     font-family: monospace;
-}
-
-#editor-console .console .log {
-    color: #FFFFFF;
+    user-select: text;
```

Comment:
With CSS prefixes, you always want to put the unprefixed version last in the order.  You do this because a browser might support both `-moz-user-select` and `user-select`, and eventually only the latter.  You want it to use the unprefixed if possible.